var/spack/repos/builtin/packages/quantum-espresso/package.py

```diff
@@ -30,6 +30,20 @@ class QuantumEspresso(Package):
     version('5.4',   sha256='e3993fccae9cea04a5c6492e8b961a053a63727051cb5c4eb6008f62cda8f335')
     version('5.3',   sha256='3b26038efb9e3f8ac7a2b950c31d8c29169a3556c0b68c299eb88a4be8dc9048')
 
+    resource(name='environ',
+            git='https://github.com/environ-developers/Environ.git',
+            tag='v1.1',
+            when='@6.4:6.4.99 +environ',
+            destination='.'
+            )
+
+    resource(name='environ',
+            git='https://github.com/environ-developers/Environ.git',
+            tag='v1.0',
+            when='@6.2.1:6.2.99 +environ',
```

Comment:
Just to be sure: didn't you mean `6.3.99` here?

Reply:
It could be, but  if you look at the [QE <-> ENV supported versions](https://github.com/environ-developers/Environ/releases), there's an overlap of QE versions between Environ 1.0 and 1.1. In order to support all combinations i'd need a variant for each Environ version e.g. `+environ1.0`, `+environ1.1` , `+environ2.0`, and so on ...
So to avoid this potential mess I locked `v1.0` to the 6.2.x series.